SEP-0002-revised.md

```diff
@@ -0,0 +1,253 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Abstract
+The SunPy project has changed significantly since the first version of SEP-0002 was written in 2014.
+This SEP details a new governance structure designed to enable the project to grow, while allowing for more input from the wider SunPy community.
+
+## Detailed Description
+The primary role of the SunPy Project is to *facilitate and promote the use and development of community-led, free and open-source solar data-analysis software based on the scientific Python environment*.
+This includes but is not limited to the following tasks:
+
+* developing the sunpy core library and other affiliated packages
+* manage and protect the SunPy brand and identity
+* promote SunPy to the community
+* managing SunPy assets, i.e. GitHub organization, domain name etc.
+
+# Governance Structure
+
+The SunPy project's governance involves the following five components:
+
+* The SunPy Community
+* The Project Members
```

Comment:
I think it'll be an unnecessary source of confusion, such as:

- Someone gets a contribution accepted into `sunpy`, and thus has participated in the open-development project that is `sunpy`, incorrectly believing that makes them be a Project Member.
- Someone mistakenly believing that only Project Members are allowed to contribute to `sunpy`.
- Someone unfamiliar with the formal definition of Project Member assuming that it is equivalent to a contributor (e.g., when reading it on another person's CV).

You could say that people need only read the definition of "Project Member", which would presumably be posted on the website, but those not in the know might not realize that need to even look up the definition.  Why not simply name the role something that would not be readily misinterpreted?  I'll point out that the analogous role in Astropy is called "Voting Member", and the adjective "voting" makes it clear that this role is distinct from the typical contributor.

All of the above would be mitigated if the role were called "Project Custodian", because "custodian" isn't readily misinterpretable as "member" is, and naturally conveys a level of responsibility.  I'll also point out that the word "custodian" is literally in current description of what a Project Member should do.

Reply:
I tend to agree with @ayshih concerns here. I like the term `Voting Member`. No one would think that they are automatically a `Voting Member` if they contributed to `sunpy`. If we went the Custodian name route I would rather we use `Project Maintainer`. It's more in keeping with other open source projects (see [Leadership and Governance](https://opensource.guide/leadership-and-governance/) and again solves the issue that no one would be confused as to whether they are a maintainer.

Reply:
I would rather not use "maintainer" as I think that is a separate role in the project for someone who is maintaining one or more code packages, where as "Project Members" could be contributing in other ways.

I have never particularly liked Astropy's choice of "Voting Member" although I will admin it is very descriptive and to the point. I do quite like "Project Custodian" as I think it's also a descriptive word while not being as narrow in scope as "Voting". (I prefer Custodian as it coveys the responsibility of the position rather than the privileges given by it.)

Reply:
So just to pointlessly weigh in; I agree that project maintainer is a separate role to governance and shouldn't be used. I also think the reason why it is listed in that github article is because they are focused on this pre-defined github roles rather than an external governance structure. For the other 3 suggestions:

I like  "project member" fine and it is straightforward which is appealing but, as many others have said, it is so general as to invite questions as to why contributing to sunpy does not make you a member or invites the assumption that only sunpy members contribute to sunpy. 

"Voting member" solves this problem with project member but it also underexpresses the point of the role as defined here, in my opinion anyway.

"Project Custodian" is probably the preferred option for me, as others have said it encompasses the role more completely. The downside is that it isn't as simple, you would rarely describe yourself as a custodian of a project in casual conversation but would describe your self as a member readily.

I mean there are countless options: "core member" which has less problems than "project member" but has confusion with sunpy core or along the same lines as custodian project facilitator/advisor/steward.

I think the reality is that these choices are all equivalent in quality.

Reply:
I think I <del>stole</del> borrowed Project Member from yt for what it's worth. https://yt-project.org/members.html

Reply:
I'm fine with "Project Custodian", which is not surprising since I suggested it. =)  I don't mind the flowery-ness.